src/main/webapp/script.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
```

Comment:
also missing tests for this file, but I think we can get by without it temporarily while we figure out a proper javascript testing framework. Please file an issue for us to figure this out.